src/utils/installExtension.ts

```diff
@@ -23,6 +21,8 @@ export async function installExtension(extensionId: string, timeoutInSeconds: nu
         if (extensions.getExtension(extensionId)) {
             return true;
         }
+        const commandToRun: string = 'extension.open';
+        commands.executeCommand(commandToRun, extensionId);
```

Comment:
Doesn't this re-open the page every 5 seconds? That sounds annoying...

Reply:
Yeah, I going to guess this isn't intentional, but this would re-open the page until they installed it every 5 seconds for a minute...

Reply:
Yeah that wasn't intentional... I'm fixing it so that it only opens the page once.